docs/use_cases/personalized_search.md

```diff
@@ -1,15 +1,108 @@
-<!-- TODO: Replace this text with a summary of article for SEO -->
+# Personalized Search: Harnessing the Power of Vector Embeddings
 
-# Personalized Search
+## Introduction
 
-<!-- TODO: Cover image: 
-1. You can create your own cover image and put it in the correct asset directory,
-2. or you can give an explanation on how it should be and we will help you create one. Please tag arunesh@superlinked.com or @AruneshSingh (GitHub) in this case. -->
+Imagine a world where your online searches return results that truly understand your needs, a world where you don't have to know the exact words to find what you're looking for. This isn't a vision of some distant possible future; it's happening now. Companies like Pinterest, Spotify, eBay, Airbnb, and Doordash have already taken advantage of the treasure trove of insights inherent in data (data that is growing exponentially, and projected to surpass 175 zettabytes by 2025 [source](https://www.forbes.com/sites/tomcoughlin/2018/11/27/175-zettabytes-by-2025)), to significantly improve user experience and engagement, conversion rates, and customer satisfaction. Spotify, for example, has been able to enhance its music recommendation system, leading to a more than 10% performance improvement in session and track recommendation tasks, and a significant boost in user engagement and satisfaction [(Hansen et al. 2020)](https://doi.org/10.1145/3383313.3412248).
 
-your content here
+And _how_ have they done this? How have these companies been able to harvest the inherent power of data to their benefit?
+
+The answer is vector embeddings. 
+
+Vector embeddings let you return more _relevant_ results to your search queries by 1) querying the _meaning_ of the search terms, as opposed to just looking for search keyword _matches_, and 2) informing your search query with the _meaning_ of personal preference data, through the addition of a personal preference vector.
+
+Let's look first at how vector embeddings improve the relevance of search query results generally, and then at how vector embeddings permit us to use the meaning of personal preferences to create truly personalized searches.
+
+![Illustration of vector embeddings](../assets/use_cases/personalized_search/embeddings.png)
```

Comment:
Hey @dolphinlog , I can't seem to find these images in the PR. Can you push them with your commit as well? Or do you want me to create them and these are dummy codes